tests/MobilePayPaymentTest.php

```diff
@@ -0,0 +1,567 @@
+use SwedbankPay\Api\Service\Payment\Resource\Collection\Item\PriceItem;
+use SwedbankPay\Api\Service\MobilePay\Request\Purchase;
+
+use SwedbankPay\Api\Service\MobilePay\Resource\Request\PaymentPayeeInfo;
+use SwedbankPay\Api\Service\MobilePay\Resource\Request\PaymentPrefillInfo;
+use SwedbankPay\Api\Service\MobilePay\Resource\Request\PaymentUrl;
+use SwedbankPay\Api\Service\MobilePay\Resource\Request\Payment;
+use SwedbankPay\Api\Service\MobilePay\Resource\Request\PaymentObject;
+
+use SwedbankPay\Api\Service\Data\ResponseInterface as ResponseServiceInterface;
+use SwedbankPay\Api\Service\Resource\Data\ResponseInterface as ResponseResourceInterface;
+
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\CreateCapture;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\CreateReversal;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\CreateCancellation;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Resource\Request\TransactionCapture;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Resource\Request\TransactionReversal;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Resource\Request\TransactionCancellation;
+
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Request\TransactionObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\AuthorizationObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\CaptureObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\ReversalObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\CancellationObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\TransactionObject as TransactionObjectResponse;
+
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetAuthorization;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetCapture;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetReversal;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetCancellation;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetTransaction;
+
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetAuthorizations;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetCaptures;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetReversals;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetCancellations;
+use SwedbankPay\Api\Service\MobilePay\Transaction\Request\GetTransactions;
+
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\AuthorizationsObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\CancellationsObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\CapturesObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\ReversalsObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\TransactionsObject;
+// phpcs:enable
+
+// phpcs:ignore
+class MobilePayPaymentTest extends TestCase
```

Comment:
Codacy found an issue: [The class MobilePayPaymentTest has a coupling between objects value of 39. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5056329)